pkg/documents/documents.go

```diff
@@ -246,6 +264,24 @@ func NewDocumentRepo(logger log.Logger, db *sql.DB) DocumentRepository {
 		logger: logger,
 	}
 }
+
+func (r *sqlDocumentRepository) findActiveDocument(customerID string, documentID string, namespace string) error {
+	query := `select documents.document_id from documents
+inner join customers on customers.namespace = ? where documents.customer_id = ? and documents.document_id = ? and documents.deleted_at is null
+limit 1;`
```

Comment:
I don't think the `limit 1` here is needed since `documentID` should be unique.

Reply:
Yea, but I don't think `limit 1` hurts anything. It does make clear the query will return one row.